Tasks/DownloadBuildArtifactsV0/task.json

```diff
@@ -9,7 +9,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 186,
+        "Minor": 187,
```

Comment:
Could you please bump it as 0.186.2 instead, and for task.loc.json file also?

Reply:
Done!